vector/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <!--TODO decide whether these themes are the right base-->
+    <style name="Theme.Vector.Light.Base" parent="@style/Theme.AppCompat.Light.NoActionBar" />
+    <style name="Theme.Vector.Dark.Base" parent="@style/Theme.AppCompat.NoActionBar"/>
+
+    <style name="Theme.Vector.Light.Common" parent="Theme.Vector.Light.Base">
```

Comment:
I took this idea of redirecting colors from k9 mail. This would require me to change every color reference into an attribute reference here. Does anyone have a better way to do this which would require changing less code? If not, it won't be too bad on my end (I can write a couple of scripts), but this would probably cause a bunch of merge conflicts...